examples/prometheus/index.js

```diff
@@ -0,0 +1,25 @@
+const {Meter} = require("@opentelemetry/metrics");
+const {PrometheusExporter} = require("@opentelemetry/exporter-prometheus");
+
+const meter = new Meter();
+
+const exporter = new PrometheusExporter({
+  startServer: true,
+}, () => {
+  console.log("prometheus scrape endpoint: http://localhost:9464/metrics")
+})
+
+meter.addExporter(exporter);
+
+const monotonicCounter = meter.createCounter("monotonic_counter", {monotonic: true, description: "monotonic counter"});
```

Comment:
Could you please add a line explaining monotonic and non-monotonic terms?

Reply:
description:  `s/monotonic counter/Example of a monotonic counter`? Same for others.

Reply:
👍